pkg/controller/performanceprofile/components/kubeletconfig/kubeletconfig.go

```diff
@@ -41,6 +42,9 @@ func New(profile *performancev2.PerformanceProfile) (*machineconfigv1.KubeletCon
 			"cpu":    defaultSystemReservedCPU,
 			"memory": defaultSystemReservedMemory,
 		},
+		FeatureGates: map[string]bool{
```

Comment:
It will not work, you will need a FeatureGate resource

Reply:
"It seems too easy" I was thinking. Thanks for pointing this out.